base/src/main/scala/co/uproot/abandon/BuildInfo.scala

```diff
@@ -0,0 +1,20 @@
+package co.uproot.abandon
+
+import scala.Predef._
+
+/** This object was generated by sbt-buildinfo. */
```

Comment:
Umm.. the point of this issue is to automate the version numbering. Developers will only update the sbt file when the version number increases, and the application will automatically pick up the version number when emitting to the XML file, etc.

If there is any manual step, then it defeats the purpose.

I suggest you verify whether the manual step is necessary.